dotcom-rendering/src/lib/useApi.tsx

```diff
@@ -16,7 +16,7 @@ const fetcher = (init?: RequestInit) => (url: string) =>
 		.then(checkForErrors)
 		.then((res) => res.json());
 
-interface ApiResponse<T> {
+export interface ApiResponse<T> {
```

Comment:
You could leave this un-exported, and have the method infer the type.

Reply:
I think our linting rules require the return type on an exported function to be explicit but I'll give it a go